Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -222,27 +228,30 @@ void IPostBackEventHandler.RaisePostBackEvent (string eventArgument)
     /// <param name="eventArgument"> The path to the clicked tree node. </param>
     private void HandleExpansionCommandEvent (string eventArgument)
     {
-      string[] pathSegments;
-      WebTreeNode? clickedNode = ParseNodePath(eventArgument, out pathSegments);
-      if (clickedNode != null)
+      WebTreeNode? clickedNode = ParseNodePath(eventArgument, out _);
+      if (clickedNode == null)
+        return;
+
+      _focusededNode = clickedNode;
+      if (clickedNode.IsEvaluated)
       {
-        _focusededNode = clickedNode;
-        if (clickedNode.IsEvaluated)
-        {
-          clickedNode.IsExpanded = !clickedNode.IsExpanded;
-          if (clickedNode.IsExpanded && EnableLookAheadEvaluation)
-            clickedNode.EvaluateChildren();
-        }
-        else
-        {
-          clickedNode.EvaluateExpand();
-          if (EnableLookAheadEvaluation)
-            clickedNode.EvaluateChildren();
-        }
+        clickedNode.IsExpanded = !clickedNode.IsExpanded;
+        if (clickedNode.IsExpanded && EnableLookAheadEvaluation)
+          clickedNode.EvaluateChildren();
+      }
+      else
+      {
+        clickedNode.EvaluateExpand();
+        var hasChildren = clickedNode.Children.Count > 0;
+        if (!hasChildren)
+          _updateMessageText = clickedNode.Text.ToString() + " " + GetResourceManager().GetString(ResourceIdentifier.ScreenReaderNoItemFoundText);
```

Comment:
clickedNode.Text + GetResourceManageer.GetText(...)

`WebString` combining: we may have examples.
@a-ctor

Reply:
Talked with @a-ctor, we do not have examples. 

IMO: 
Adding a combiner for `WebString`s would be overkill, and we need the end output of `_updateMessageText` to be a string anyways, unless there's some way to put webstring directly into TS